library/core/src/primitive_docs.rs

```diff
@@ -1160,9 +1160,9 @@ impl<T> (T,) {}
 ///
 /// Note that most common platforms will not support `f16` in hardware without enabling extra target
 /// features, with the notable exception of Apple Silicon (also known as M1, M2, etc.) processors.
-/// Hardware support on x86-64 requires the avx512fp16 feature, while RISC-V requires Zhf.
-/// Usually the fallback implementation will be to use `f32` hardware if it exists, and convert
-/// between `f16` and `f32` when performing math.
+/// Hardware support on x86/x86-64 requires the avx512fp16 or avx10.2 features, while RISC-V requires
+/// Zfh, and Arm/AArch64 requires FEAT_FP16.  Usually the fallback implementation will be to use `f32`
```

Comment:
Are there actually 32-bit x86 processors with the listed avx features?

`FEAT_FP16` should probably be in ticks `` `...` ``

Reply:
Yes, they’re called x86-64 processors in 32-bit mode.  (And yes, AVX-512 is accessible from that mode, with some restrictions on which registers are visible.)

Reply:
Fair enough, thanks for the confirmation. Please squash then lgtm.